scripts/package-appx.mjs

```diff
@@ -0,0 +1,93 @@
+	process.exit( 1 );
+}
+
+const __dirname = path.dirname( fileURLToPath( import.meta.url ) );
+
+const windows10SDKVersionPath = path.resolve( __dirname, '..', '.windows-10-sdk-version' );
+try {
+	await fs.access( windows10SDKVersionPath );
+} catch {
+	console.error( `Windows version defintion not found at ${ windows10SDKVersionPath }.` );
+	process.exit( 1 );
+}
+const windows10SDKVersionContent = await fs.readFile( windows10SDKVersionPath );
+const windows10SDKVersion = windows10SDKVersionContent.toString().trim();
+const windowsKitPath = `C:\\Program Files (x86)\\Windows Kits\\10\\bin\\10.0.${ windows10SDKVersion }.0\\x64`;
+
+console.log( 'Verifying Windows 10 SDK location...' );
+try {
+	await fs.access( windowsKitPath );
+	console.log( `Windows 10 SDK verions ${ windows10SDKVersion } found. Continuing...` );
+} catch {
+	console.error(
+		`Windows Kit not found at ${ windowsKitPath }. Please install the Windows 10 SDK using:\n\n\t.\\.buildkite\\commands\\install-windows-10-sdk.ps1`
+	);
+	process.exit( 1 );
+}
+
+const packageJsonPath = path.resolve( __dirname, '..', 'package.json' );
+const packageJsonText = await fs.readFile( packageJsonPath, 'utf-8' );
+const packageJson = JSON.parse( packageJsonText );
+
+const outPath = path.join( __dirname, '..', 'out' );
+const assetsPath = path.join( __dirname, '..', 'assets', 'appx' );
+
+const normalizeWindowsVersion = ( version ) => {
+	const noPrerelease = version.replace( /-.*/, '' );
+	return `${ noPrerelease }.0`;
+};
+
+const appStoreVersion = normalizeWindowsVersion( packageJson.version );
+
+const appxName = packageJson.productName + '-appx';
+
+const sharedOptions = {
+	containerVirtualization: false,
+	inputDirectory: path.resolve( outPath, 'Studio-win32-x64' ),
+	packageVersion: appStoreVersion,
+	// Results in Id being invalid (might just be a matter of escaping, though)
+	// packageName: 'Studio by WordPress.com',
+	packageName: 'Studio',
+	packageDescription: packageJson.description,
+	packageExecutable: `app/${ packageJson.productName }.exe`,
+	windowsKit: windowsKitPath,
+	deploy: false,
+	assets: assetsPath,
+	makePri: true,
+	packageDisplayName: 'Studio by WordPress.com',
+	// See details at https://partner.microsoft.com/en-us/dashboard/products/<id>/identity
+	publisher: 'CN=E2E5A157-746D-4B04-9116-ABE5CB928306',
+	publisherDisplayName: 'Automattic, Inc.',
+	identityName: '22490Automattic.StudiobyWordPress.com',
+};
+
+const appxOutputPathUnsigned = path.resolve( outPath, `${ appxName }-unsigned` );
+console.log(
+	`~~~ Creating unsigned .appx for Microsoft Store submission upload at ${ appxOutputPathUnsigned }...`
+);
+
+await convertToWindowsStore( {
+	...sharedOptions,
+	devCert: 'nil', // skip code signing for Store upload
+	outputDirectory: appxOutputPathUnsigned,
+} );
+
+const appxOutputPathSigned = path.resolve( outPath, `${ appxName }-signed` );
+console.log( `~~~ Creating signed .appx for local testing at ${ appxOutputPathSigned }...` );
```

Comment:
If one tries to launch the unsigned AppX, Windows refuses to enable the "Install" button:

![image](https://github.com/user-attachments/assets/44096e32-cdf8-4c83-aeee-ecf20721f8fa)

The flow with the signed version still results in multiple warnings, but clicking through them one can eventually run the app:

<img width="899" alt="image" src="https://github.com/user-attachments/assets/5ee9a77b-e4b5-4647-afc5-6e9042f82778">

I figured given the packaging process is relatively fast, it might be useful to generate both versions just in case we need to test something.

Alternatively, we can abstract this logic and generate the unsigned AppX only when building for release.

Reply:
Is there something we could do to avoid the extra alerts?

> Alternatively, we can abstract this logic and generate the unsigned AppX only when building for release.

If it's not a big effort , limiting the Appx only for releases would be ideal. Thanks for the suggestion!

Reply:
@sejas this warning suppose to alert Mac users who run Intel binary on Apple Silicon chip, and I think it shouldn't pop up on Windows. Should we check it?

Reply:
Created the issue to investigate it further:
https://github.com/Automattic/dotcom-forge/issues/9846

The message is indeed launched by Studio: https://github.com/Automattic/studio/blob/335c634a0b7c37414f74e6eda86a7ddffe1ecb24/src/renderer.ts#L85-L102